backend/portal/api/enrichment.py

```diff
@@ -23,3 +35,17 @@ def enrich_dataset_with_ancestors(dataset, key, ontology_mapping):
     unique_ancestors = list(OrderedDict.fromkeys(flattened_ancestors))
     if unique_ancestors:
         dataset[f"{key}_ancestors"] = unique_ancestors
+
+
+def generate_tagged_ontology_id(tissue):
```

Comment:
nit: what do you think of renaming this function `generate_tagged_tissue_ontology_id` since its tissue-specific?

Reply:
Hey @nayib-jose-gloria, I have updated the function name!